packages/test/src/test-integration-update-interceptors.ts

```diff
@@ -1,6 +1,20 @@
-import { WorkflowStartUpdateInput, WorkflowStartUpdateOutput, WorkflowUpdateStage } from '@temporalio/client';
+import { randomUUID } from 'crypto';
+import {
+  WithStartWorkflowOperation,
+  WorkflowStartUpdateInput,
+  WorkflowStartUpdateOutput,
+  WorkflowStartUpdateWithStartInput,
+  WorkflowStartUpdateWithStartOutput,
+  WorkflowUpdateStage,
+} from '@temporalio/client';
 import * as wf from '@temporalio/workflow';
-import { Next, UpdateInput, WorkflowInboundCallsInterceptor, WorkflowInterceptors } from '@temporalio/workflow';
+import {
+  Next,
+  UpdateInput,
+  WorkflowIdConflictPolicy,
```

Comment:
Unused import

Reply:
Thanks, removed. (I'm used to `ruff` removing them on save, and also would expect a linter violation.)

Reply:
Maybe you are just missing some local config in vscode. I get linter errors here, and can choose either to auto-remove extra imports on save (I used to do that, but now prefer to keep it disabled)